design/overview.md

```diff
@@ -81,43 +82,36 @@ And provide a security equivalent to or greater than IBC.
-  * After some number of blocks transactions the`Peggy Bridge Tx pool` should be removed from the pool and refunded. To be re-submitted with a higher fee. 
-  * TODO: consider a gas price oracle which would allow us to better estimate profitible batches 
-  * TODO: consider allowing users to remove their MsgSendToEth transactions from the `Peggy Bridge Tx pool` at their own request, without waiting for the timeout, this should always be safe because only transactions in the `Peggy Bridge Batch pool` are even possible to submit.
+- Someone (permissionlessly) sends MsgSendToEth this creates an entry in the 'peggy bridge tx pool' this pool is in the Cosmos chain store and contains all transactions not yet bridged. Funds of the specified demon are then burned from the users account if that demon is valid to bridge, otherwise the tx is rejected.
+- Someone (permissionlessly) sends a MsgRequestBatch, this produces a new `Transaction batch` in the `Peggy Bridge Tx pool`. The creation of this batch occurs in ComsosSDK and is entierly deterministic, ordering transactions from highest to lowest fee and including them in the `Transaction batch` until there are no more transactions or the batch size limit is reached.
+- The `Orchestrators` view the `Peggy Bridge Tx pool` via REST endpoints. When a new batch appears they sign the RLP-encoded batch and submit a MsgConfirmBatch signed with their Cosmos Key that contains their Ethereum signature over the batch.
+- Once enough signatures are collected and the batch is `Confirmed` any `Orcestrator` can collect these signatures from the REST endpoint, assemble an Ethereum transaction and submit the batch.
+- The `Orchestrators` observe the Ethereum blockchain via their trusted Ethereum node. Once the batch has entered the blockchain and `EthBlockDelay` has elapsed an `Orchestrator` will send MsgBatchInChain. Containing a signed attestation that the block has entered the Ethereum chain, once the batch is `Observed` by sending a MsgBatchInChain the Cosmos -> Ethereum bridge process is complete. The batch is removed from the `Peggy Bridge Tx pool` and the MsgSendToEth flow is now fully complete.
+- It may be possible for a `TransactionBatch` to be created and `Confirmed` that is not profitible to submit. This can occur when gas prices are high or the average fee in the bridge is low. In this case the batch will either wait long enough for the gas situation to change or a new MsgRequestBatch will be greated. Exact behavior here will have to take into account a few questions 'can the previous batch be submitted? (check existing signatures in the chain)', 'is the fee to transaction ratio in the new block higher than the old block?' and 'Was the previous batch full or partially empty?'
```

Comment:
Can you separate this point (and all below) into a separate list. This list mixes clear actions as well as hypothetical design questions. I would propose something like:

```markdown
- The `Orchestrators` observe the Ethereum blockchain via their trusted Ethereum node. Once the batch has entered the blockchain and `EthBlockDelay` has elapsed an `Orchestrator` will send MsgBatchInChain. Containing a signed attestation that the block has entered the Ethereum chain, once the batch is `Observed` by sending a MsgBatchInChain the Cosmos -> Ethereum bridge process is complete. The batch is removed from the `Peggy Bridge Tx pool` and the MsgSendToEth flow is now fully complete.

Open Questions:

- It may be possible for a `TransactionBatch` to be created and `Confirmed` that is not profitible to submit. This can occur when gas prices are high or the average fee in the bridge is low. In this case the batch will either wait long enough for the gas situation to change or a new MsgRequestBatch will be greated. Exact behavior here will have to take into account a few questions 'can the previous batch be submitted? (check existing signatures in the chain)', 'is the fee to transaction ratio in the new block higher than the old block?' and 'Was the previous batch full or partially empty?'
```